release-github/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Release to GitHub"
+description: "Release library artifacts to GitHub release page"
+
+inputs:
+  library-name:
+    description: 'Name of the Python library'
+    required: true
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+  
+    - name: "Download the library artifacts from build-library step"
+      uses: actions/download-artifact@v3
+      with:
+        name: ${{ inputs.library-name }}-artifacts
+        path: ${{ inputs.library-name }}-artifacts
+
+    - name: "Display the structure of downloaded files"
+      shell: bash
+      run: ls -R
+
+    - name: "Release to GitHub"
+      uses: softprops/action-gh-release@v1
+      with:
+        files: |
+          ${{ inputs.library-name }}/**/*.whl
+          ${{ inputs.library-name }}/**/*.tar.gz
```

Comment:
We should also add the docs as part of the release (both in HTML and PDF version if possible...). That will force people to upload the documentation to the artifacts as a ZIP, and then download it here as well and push it to the release. What do you think? This may be left for a second round anyway - not a priority at all.

Reply:
Oh, that's true! Good catch. So far, those have only been uploaded as artifacts during the build.

Reply:
We can fix in this same pull-request, no worries.

Reply:
I updated the actions so PDF doc gets generated and uploaded, see https://github.com/pyansys/actions/pull/14/commits/8ba021f810f460f2a056525ac925cdffb7007f47.